samples/credentials.md

```diff
@@ -0,0 +1,97 @@
+# Set up credentials and connect to Azure Devops
+
+Before you can run Azure devops commands, you need to run login command to sign in or set up credentials. There are a few ways to establish credentials for use in the CLI.
+
+
+## Option 1: Login with Azure CLI credentials
+All MSA and AAD backed accounts can directly run `az login` command to use azure devops resources.
```

Comment:
Yes this is for organization.

Reply:
Yes, this is organization. However, I am leaving this intricacy out as it might lead to more distraction - "is my organization MSA/AAD backed?", "How do I know my organization type?"

The expectation is that users run through the Quick start steps. In the event there is a failure, they refer to the Get started guide for the detailed set up instructions.